src/color/test/getLuminance.test.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import getLuminance from '../getLuminance'
+
+describe('getLuminance', () => {
```

Comment:
Would like to see the tests for all color spaces if possible (rgb, hsl, hsla, and a named CSS color) if we could.